spec/pro_motion/data_table_screen_spec.rb

```diff
@@ -1,68 +1,107 @@
 describe 'DataTableScreen' do
-  class TestModel
-    attr_accessor :name
 
-    def initialize(name)
-      self.name = name
-    end
+  class TestDataTableScreenScope < ProMotion::DataTableScreen
+    model Contributer, scope: :starts_with_s
+  end
 
-    def self.all
-      [
-        new('one'),
-        new('two')
-      ]
-    end
+  class TestDataTableScreen < ProMotion::DataTableScreen
+    model Contributer
+  end
 
-    def self.starts_with_o
-      [
-        new('one')
-      ]
-    end
+  def contributors
+    %w{twerth squidpunch GantMan shreeve chunlea markrickert}
+  end
 
-    def cell
-      {
-        cell_class: TestCell,
-        properties: {
-          name: self.name
-        }
-      }
+  def init_contributors
+    Contributer.destroy_all
+    contributors.each do |c|
+      Contributer.new(name: c)
     end
```

Comment:
I assume this is to clear out any old test data.  I'd recommend using cdq.reset! instead if you can.